### PR TITLE
New changes, B&C checkboxes with/without spots

### PR DESCRIPTION
**Pull request by JOHNFLEURIMOND**

### CODE REVIEW

File: services-js/commissions-app/src/pages/commissions/apply.tsx [hunk 1]
```diff
@@ -36,7 +36,7 @@ export default class ApplyPage extends React.Component<Props> {
       >
         <Checkbox
           name={`commissionIds.${commission.id}`}
-          value={commission.id.toString()}
+          value={commission.id.toString() && commission.openSeats.toString()}
```
Comment (fionawhim) [inline services-js/commissions-app/src/pages/commissions/apply.tsx]: `&&` is going to evaluate to the last expression. So now the `value` for the `<input type="checkbox" />`elements is going to be the number of open seats, not the commission ID.

What was your thought process for putting this in?

File: services-js/commissions-app/src/pages/commissions/apply.tsx [hunk 2]
```diff
@@ -54,6 +54,10 @@ export default class ApplyPage extends React.Component<Props> {
 
   render() {
     const { commissions } = this.props;
+    const commissionsWithoutOpenSeats = commissions.filter(
+      commission => commission.openSeats
```
Comment (fionawhim) [inline services-js/commissions-app/src/pages/commissions/apply.tsx]: This is a good start, but double-check your logic in this filter function. `commission.openSeats` is going to be true-ish when it's not 0.

File: services-js/commissions-app/src/pages/commissions/apply.tsx [hunk 3]
```diff
@@ -327,8 +326,32 @@ export default class ApplyPage extends React.Component<Props> {
                   error={touched.otherInformation && errors.otherInformation}
                   onBlur={handleBlur}
                 />
+
                 <hr className="hr hr--sq" />
                 <SectionHeader title="Boards and Commissions" />
+                <h2>
+                  "Please note that many of these Boards and Commissions require
```
Comment (fionawhim) [inline services-js/commissions-app/src/pages/commissions/apply.tsx]: You can remove the `"` from around this sentence.

**Review by fionawhim — CHANGES_REQUESTED** [review]

File: services-js/commissions-app/src/pages/commissions/apply.tsx (hunk 1 above)
Comment (JOHNFLEURIMOND) [reply services-js/commissions-app/src/pages/commissions/apply.tsx]: This was supposed to be taken out. This was in the beginning stage where I thought I had to incorporate that.

**Review by fionawhim — APPROVED** [review]
Looks good!

**Pull request merged**